src/test/kotlin/nl/info/zac/productaanvraag/ProductaanvraagEmailServiceTest.kt

```diff
@@ -87,6 +90,62 @@ class ProductaanvraagEmailServiceTest : BehaviorSpec({
         }
     }
 
+    Given("zaak created from productaanvraag and automatic email is enabled with GEEMENTE as sender") {
```

Comment:
And GEMEENTE as `replyTo` address? Would be good to test that as well here. Maybe in a separate test but that's up to you.

Reply:
So did you do this? Why did you close the conversation?